File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int h;
    do
    {
        // pyramid height query
        h = get_int("Height: ");
    }
    while (h < 1 || h > 8);

    // for each row
    for (int i = 0; i < h; i++)
    {

        // for each column
        for (int j = 0; j < 2 * h + 2; j++)
        {
            //print а "space"
            if (j < h - 1 - i)
            {
                printf(" ");
            }
            //print brick
            else if (j < h)
            {
                printf("#");
            }
            //print double spase
            else if (j < h + 2)
            {
                printf(" ");
            }
            //print last brick
            else if (j < h + 3 + i)
            {
                printf("#");
            }
        }
        //move to next row
        printf("\n");
    }
}

File: sentimental-mario-more/mario.py
from cs50 import get_int

while True:
    h = get_int("Heyght: ")
    if h > 0 and h < 9:
        break

for i in range(h):
    for j in range(2 * h + 2):

        # print а "space"
        if j < (h - 1 - i):
            print(" ", end="")

        # print brick
        elif j < h:
            print("#", end="")

        # print double spase
        elif j < (h + 2):
            print(" ", end="")

        # print last brick
        elif j < (h + 3 + i):
            print("#", end="")
    print()
